sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/DocumentModelDetails.Serialization.cs

```diff
@@ -7,7 +7,7 @@ public partial class DocumentModelDetails
     {
         /// <summary> Deserializes the model from a raw response. </summary>
         /// <param name="response"> The response to deserialize the model from. </param>
-        internal static new DocumentModelDetails FromResponse(Response response)
+        internal static DocumentModelDetails FromResponse(Response response)
```

Comment:
The service removed the parent class `DocumentModelSummary`, so we don't need the `new` keyword anymore.